src/screens/TopicMemberScreen/index.js

```diff
@@ -88,15 +88,15 @@ const TopicMemberScreen = () => {
     const result = await getAllMemberTopic(query);
     if (result.code === 200) {
       const newDataFollowed = result.data
-        .filter((item) => item.is_following)
+        .filter((item) => item.is_following && profile.myProfile.user_id !== item.user_id)
         .map((data) => ({
           ...data,
           name: data.username,
           image: data.profile_pic_path,
           description: null
         }));
       const newDataUnfollowed = result.data
-        .filter((item) => !item.is_following)
+        .filter((item) => !item.is_following && profile.myProfile.user_id !== item.user_id)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the current user is also excluded from the unfollowed list. This maintains consistency in the application's behavior.

```diff
- .filter((item) => !item.is_following)
+ .filter((item) => !item.is_following && profile.myProfile.user_id !== item.user_id)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->